src/analyzer/protocol/ldap/ldap.spicy

```diff
@@ -267,13 +264,18 @@ public type Message = unit {
     ProtocolOpcode::INTERMEDIATE_RESPONSE   -> INTERMEDIATE_RESPONSE:   NotImplemented(self);
     ProtocolOpcode::MOD_DN_REQUEST          -> MOD_DN_REQUEST:          NotImplemented(self);
     ProtocolOpcode::SEARCH_RESULT_REFERENCE -> SEARCH_RESULT_REFERENCE: NotImplemented(self);
-  } &parse-from=self.applicationBytes if ( self.opcode );
+  } &size=self.protocolOp.len.len;
 
-  on %error {
+  # Eat the controls field if it exists.
+  : skip bytes &size=self.seqHeader.len.len - (self.offset() - self.seqHeaderLen);
```

Comment:
Are there ways where we could genuinely see the subtraction underflow? If we could let's instead use `&size=0` here (checking requires casting the values to `int64`); if we cannot see that let's maybe add explicit `&requires` checking the values as soon we have them available since for a user "integer underflow" looks like a bug, but `&requires failed` does less.

Reply:
> Are there ways where we could genuinely see the subtraction underflow?

Not genuinely, don't think so. Let me put a `&requires`.